client/src/pages/Paste.js

```diff
@@ -49,6 +63,7 @@ function Paste({ match, embedded }) {
           <Author name={paste.author} />
           <PastaPoints pastaPoints={paste.pastaPoints} pasteId={pasteId} />
           <PasteArea>{paste.value}</PasteArea>
+          <ReportButton onClick={handleReport}>Report that shit!</ReportButton>
```

Comment:
It's possible to send multiple reports.
Maybe it makes sense to use the `Confirmation` component